Apps/Fences/Fences 5/zh_CN.po

```diff
@@ -2487,86 +2485,86 @@ msgid "Welcome"
 msgstr "欢迎"
 
 msgid "Welcome to Fences®"
-msgstr "欢迎来到桌面分区"
+msgstr "欢迎来到 Fences"
 
 msgid "What is kiosk mode?"
-msgstr "什么是信息亭模式？"
+msgstr "什么是 Kiosk 模式？"
 
 msgid "When I plug in a new primary monitor, move my icons to it."
-msgstr "当我插上一台新的主显示器时，将我的图标移动到其上。"
+msgstr "当我插入一台新的主显示器时，将我的图标自动转移到其上"
 
 msgid ""
 "When you installed Fences, a snapshot of your desktop's layout was taken.\n"
 "\n"
 "Would you like to restore your desktop icons' positions to before Fences was installed?"
 msgstr ""
-"当您安装桌面分区时，会拍摄桌面布局的快照。\n"
+"当您安装 Fences 时，程序自动创建了一个当时桌面布局的快照。\n"
 "\n"
-"您是否希望将桌面图标的位置恢复到安装桌面分区之前的位置？"
+"您是否希望将桌面图标的位置恢复到安装 Fences 之前的位置？"
 
 msgid "Where did my icons go?"
-msgstr "我的图标去了哪里？"
+msgstr "我的图标哪里去了？"
 
 msgid "Where did my icons go? No need to worry."
-msgstr "我的图标去了哪里？无需担心。"
+msgstr "我的图标哪里去了？它们只是折叠起来了。"
 
 #, fuzzy
 msgid "With Fences you can group icons on your desktop, access folders from your desktop, and much more. Click the features on the left to learn more about what Fences can do."
-msgstr "有了 Fences，你可以对桌面上的图标进行分组，从桌面访问文件夹，等等。点击左侧的功能，了解有关 Fences 功能的更多信息。"
+msgstr "有了 Fences，您可以对桌面上的图标进行分组，从桌面访问其他路径的文件夹……点击左侧的面板，了解有关 Fences 功能的更多信息。"
 
 #, fuzzy
 msgid "With Rollup, you can collapse a Fence into its titlebar to keep your icons tucked away."
-msgstr "使用 “卷轴” 功能，你可以将栅栏折叠到标题栏中，使图标隐藏起来。"
+msgstr "使用“卷折”功能，您可以将桌面分区折叠到标题栏中，使图标隐藏起来。"
 
 #, fuzzy
 msgid "With Vanish, you can quickly hide && show your desktop icons by double-clicking blank space on your desktop."
-msgstr "有了 Vanish，你只需双击桌面上的空白处，就能快速隐藏和显示桌面图标。"
+msgstr "启用“速隐功能”后，您只需双击桌面上的空白处，就能快速隐藏和显示桌面图标。"
 
 msgid "Work better with Fences"
-msgstr "借助桌面分区更好地完成工作"
+msgstr "Fences，静·净之选"
 
 #, fuzzy
 msgid ""
 "Would you like to disable showing Fences in the taskbar?\n"
 "\n"
 "You can re-enable it later from Fences Settings in the Quick Access area."
 msgstr ""
-"您想禁止在任务栏中显示栅栏吗？\n"
+"您想禁止在任务栏中显示 Fences 吗？\n"
 "\n"
-"您可以稍后通过快速访问区域中的 \"Fences设置 \"重新启用。"
+"您可以稍后通过快速访问区域中的 Fences 设置 重新启用它"
 
 msgid "Would you like to take a snapshot of your current configuration before applying auto-layout?"
-msgstr "您是否要在应用自动布局之前拍摄当前配置的快照？"
+msgstr "您是否希望在应用自动布局之前创建当前配置的快照？"
 
 msgid "Would you like to take a snapshot of your current configuration before loading this snapshot?"
-msgstr "您是否要在加载此快照之前拍摄当前配置的快照？"
+msgstr "您是否要在加载此快照之前创建当前配置的快照？"
 
 msgid "Would you like to take a snapshot of your current configuration before performing an auto-layout?"
-msgstr "您是否要在执行自动布局之前拍摄当前配置的快照？"
+msgstr "您是否要在执行自动布局之前创建当前配置的快照？"
 
 msgid "Yes"
 msgstr "是"
 
 msgid "You can change the size of icons in a Folder Portal by selecting an option from its \"View\" right-click menu, or by clicking anywhere on it, and then holding control while using your mouse's scroll-wheel."
-msgstr "您可以通过从其“视图”右键菜单中选择一个选项，或通过单击任意位置，然后按住控件的同时使用鼠标滚轮，以更改Folder Portal中的图标的大小。"
+msgstr "您可以通过从在“视图”右键菜单中选择一个选项；或单击文件夹的任意位置，然后按住 Ctrl 并滑动鼠标滚轮，以更改 映射文件夹 中的图标的大小"
```

Comment:
@Mikachu2333 'Folder Portal' is considered a reserved phrase (a Trademarked one at that).  It needs to be preserved in the translation.

@jeffbargmann

Reply:
OK, I only made a “free” translation for the convenience of Chinese users, but now I will correct it.